src/Configuration/ShowViewConfigPass.php

```diff
@@ -20,7 +24,12 @@ class ShowViewConfigPass implements ConfigPassInterface
         'embedded_list' => '@EasyAdminExtension/default/field_embedded_list.html.twig',
     );
 
-    public function __construct($embeddedListHelper)
+    /**
```

Comment:
I don't like type hinting with class, it reduces extensivity. The best approach, ihmo, is to type hint with interfaces. For now, project is too young to define those interfaces, so I prefer no type hint at all for services constructor arguments.

Please revert those changes.

Reply:
reverted